lib/mixins/strict-binding-parser.html

```diff
@@ -0,0 +1,405 @@
+                  state = STATE.FIRSTCLOSINGBINDING;
+                  break;
+                }
+                case '\'':
+                case '"': {
+                  quote = char;
+                  state = STATE.STRING;
+                  break;
+                }
+                case '(': {
+                  bindingData.signature = {
+                    methodName: text.substring(bindingData.startChar, i).trim(),
+                    args: [],
+                    static: true
+                  };
+                  bindingData.startChar = i + 1;
+                  state = STATE.METHOD;
+                  break;
+                }
+                case ':': {
+                  state = STATE.FIRSTCOLON;
+                }
+              }
+              break;
+            }
+            case STATE.FIRSTCOLON: {
+              if (char === ':') {
+                bindingData.customEvent = true;
+                bindingData.startCharAfterColon = i + 1;
+                state = STATE.COLONNOTIFYEVENT;
+              } else {
+                state = STATE.BINDING;
+              }
+              break;
+            }
+            case STATE.COLONNOTIFYEVENT: {
+              if (char === BINDINGS[bindingData.mode]) {
+                state = STATE.COLONNOTIFYEVENTFIRSTCLOSINGBINDING;
+              }
+              break;
+            }
+            case STATE.COLONNOTIFYEVENTFIRSTCLOSINGBINDING: {
+              if (char === BINDINGS[bindingData.mode]) {
+                bindingData.event = text.substring(bindingData.startCharAfterColon, i - 1).trim();
+                const prop = text.substring(bindingData.startChar, bindingData.startCharAfterColon - 2).trim();
+                storeVariableBinding(parts, bindingData, prop, i);
+                state = STATE.INITIAL;
+              } else {
+                state = STATE.BINDING;
+              }
+              break;
+            }
+            case STATE.FIRSTCLOSINGBINDING: {
+              if (char === BINDINGS[bindingData.mode]) {
+                const prop = text.substring(bindingData.startChar, i - 1).trim();
+                storeVariableBinding(parts, bindingData, prop, i);
+                state = STATE.INITIAL;
+              } else {
+                state = STATE.BINDING;
+              }
+              break;
+            }
+            case STATE.STRING: {
+              if (char === '\\') {
+                escaped = true;
+              } else if (char === quote && !escaped) {
+                state = STATE.BINDING;
+              } else {
+                escaped = false;
+              }
+              break;
+            }
+            case STATE.METHOD: {
+              switch (char) {
+                case ')': {
+                  storeMethodVariable(bindingData, text, i);
+                  storeMethod(bindingData, templateInfo);
+                  bindingData.startChar = i + 1;
+                  storeMethod(bindingData, templateInfo);
```

Comment:
Is this a typo to call `storeMethod` a second time here? If so, let's fix and add a test to verify.

Reply:
@TimvdLippe acknowledged issue and will fix.

Reply:
I no longer have write access to this repository, so I submitted #4866 on this branch to fix the issue